guestbook/views.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.contrib import messages
+from django.shortcuts import render
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.views.decorators.http import require_http_methods
+
+from .models import Comment
+
+class CommentForm(forms.ModelForm):
```

Comment:
You would normally move this form into `forms.py`, even if that's the only form in that app. It keeps things clear and well structured.